src/Models/QuoteItemOption.php

```diff
@@ -57,7 +58,7 @@ protected function label(): Attribute
     protected function option(): Attribute
     {
         return Attribute::make(
-            get: fn () => config('rapidez.models.product_option')::find(explode('_', $this->code)[1])
+            get: fn () => config('rapidez.models.product_option')::find(explode('_', $this->code)[1] ?? '')
```

Comment:
I think this is a better fallback
```suggestion
            get: fn () => config('rapidez.models.product_option')::find(explode('_', $this->code)[1] ?? $this->code)
```